docs/learn/dapp-staking/dapp-staking-protocol.md

```diff
@@ -282,16 +282,20 @@ The comparison is always done between the _baseline_ number of slots and the new
 
 $new\_threshold = base\_threshold * (1 + \Delta\%_{threshold})$
 
-At the moment, there are two types of tier entry thresholds:
+There are now two types of tier entry thresholds:
 
-* `Dynamic` - adjusts the threshold based on the aforementioned formula.
-* `Fixed` - defines a static, fixed threshold which doesn't adapt.
+* `Dynamic` - A percentage of the total issuance as staked funds that can change between periods. It includes a minimum percentage that the threshold cannot fall below. This type is used for _higher_ tiers.
+* `Fixed` - A constant percentage of the total issuance as staked funds, which does not change between periods. Used for the _lowest_ tier, and defines a static value.
 
-The `Dynamic` threshold is used for _higher_ tiers, and also defines the minimum amount to which the threshold can fall.
-E.g. a threshold might define _current_ value as **1,000,000 ASTR**. Since this can be decreased if number of slots goes up,
-it also defines a minimum amount the threshold can take, e.g. **500,000 ASTR**.
+These percentages are calculated based on a total issuance of **8.4 billion ASTR** tokens when dApp Staking V3 was launched. As the total issuance changes (e.g. burn events), the dynamic thresholds will adjust accordingly, ensuring a fair and adaptive staking environment. However, the fixed threshold for the *lowest* tier remains constant.
```

Comment:
Maybe it's a bit misleading to say the last part, that the fixed threshold remains a constant. It's still a percentage, that moves with total issuance.

Reply:
I agree, I just removed it.